models/models.go

```diff
@@ -99,9 +105,33 @@ func (ep *ExportPayload) SetStatusComplete(t *time.Time, s3key string) {
 	ep.S3Key = s3key
 }
 
-func (ep *ExportPayload) SetStatusFailed()  { ep.Status = Failed }
+func (ep *ExportPayload) SetStatusPartial(t *time.Time, s3key string) {
+	ep.Status = Partial
+	ep.CompletedAt = t
+	ep.S3Key = s3key
+}
+
+func (ep *ExportPayload) SetStatusFailed() {
+	t := time.Now()
+	ep.Status = Failed
+	ep.CompletedAt = &t
```

Comment:
Should we set a complete time stamp for a total failure?

Reply:
Seems okay to me to do that, it just marks that processing of this payload is complete, even though it went poorly